src/main/java/bio/terra/stairway/Queue.java

```diff
@@ -62,47 +67,65 @@
   private final Stairway stairway;
   private final Publisher publisher;
   private final String subscriptionName;
-  private SubscriberStub subscriber;
+  private SubscriberStub subscriberStub;
 
-  public Queue(Stairway stairway, String projectId, String subscriptionId, String topicId)
+  public Queue(Stairway stairway, String projectId, String topicId, String subscriptionId)
```

Comment:
Any particular reason for the reorder? Is it due to the hierarchy of project -> topic -> subscription?

Reply:
If I  recall correctly, i had used different orders for different methods. this order made more sense to me because it is in the right hierarchy.